tests/e2e/rudder-npm-js/README.md

```diff
@@ -0,0 +1,26 @@
+# Getting Started with Rudder Typer
+
+## Prerequisite(If you want to use your own tracking plan)
```

Comment:
Additionally, they'd need to adjust the rudder typer instrumentation as per their tracking plan.

Reply:
done